### ISSUE TITLE
Exception: must be string or buffer, not None

### ISSUE THREAD

**Issue by BenJoParadise** [ecederstrand/exchangelib#436]
https://github.com/ecederstrand/exchangelib/blob/8adaee98f6227aa0de199a362d8bb9c96e51cf57/exchangelib/attachments.py#L165

When getting an empty file, this line throws exception (the decode function, not the .text)

Reproduction:
1. Create empty file (i used `touch`)
2. Attach the file to an email message (don't use OWA, it blocks this behaviour)
3. Send the message
4. Try to retrieve the attachment.content using the EWS API

**Comment by ecederstrand** [comment]
Thanks! That should be easy to test and fix.

**Comment by ecederstrand** [comment]
Hmm, I can't reproduce this. My server just returns this XML:

```xml
            <t:FileAttachment>
              <t:AttachmentId Id="AAMkADQyYzZmYmUxLTJiYjItNDg2Ny1iMzNjLTIzYWE1NDgxNmZhNABGAAAAAADUebQDarW2Q7G2Ji8hKofPBwAl9iKCsfCfSa9cmjh+JCrCAAPJcuhjAABioKiOUTCQRI6Q5sRzi0pJAAFVx1r9AAABEgAQACAY/CckzfNLvGgvQvDc6II="/>
              <t:Name>empty_file.txt</t:Name>
              <t:ContentType>text/plain</t:ContentType>
              <t:Content/>
            </t:FileAttachment>
```

which is parsed without problems. `attachment.content` becomes `None` on the returned attachment, which is still an error, but I don't get the exception you got. My test code is this:

```python
item = Message(folder=account.inbox, subject='Empty attachment test')
att1 = FileAttachment(name='empty_file.txt', content=b'')
item.attach(att1)
item.save()
fresh_item = list(account.fetch(ids=[item]))[0]
print(fresh_item.attachments[0].content)
```

**Comment by ecederstrand** [comment]
Oh. This is actually a duplicate of #419, fixed in a6b197a97. Closing.
